roles/grafana-repo/files/grafana.repo

```diff
@@ -0,0 +1,9 @@
+[grafana]
+name=grafana
+baseurl=https://packagecloud.io/grafana/stable/el/6/$basearch
+repo_gpgcheck=1
```

Comment:
Thanks for the observation. I will check the issue and add a workaround into this pull request.